### DISCUSSION TITLE
Payment Gateways transaction Fee in the Givewp

### DISCUSSION THREAD

**Discussion by ashokngi** [impress-org/givewp#5865] [Developer Q&A] [answered]
Hi,
There is no Payment Gateways transaction Fee in the Givewp, Like when a transaction is processed, there is a processing cost of the payment gateway that we need to display somewhere at Wordpress admin Givewp dashboard,
Someone please guide how to get the Transaction Fee (Processing cost)
Attached is Stripe interface for transaction fee
![Stripe](https://user-images.githubusercontent.com/86282728/124745220-054d1600-df39-11eb-8b43-c591a39ad4d5.png)

**Comment by JasonTheAdams** [comment, marked as answer]
Hi @ashokngi!

This isn't really a development question, so Github isn't the best place to get this answered. If, however, you use our [contact form](https://givewp.com/contact-us/) or simply email customers@givewp.com, our Customer Success team would be happy to help get your question answered!
